app/pages/HomePage/components/GuideList/GuideList.jsx

```diff
@@ -12,33 +13,57 @@ import ImgYouthHomelessnessPartner from './assets/logos/larkin.png';
 import ImgAdultHomelessness from './assets/AdultHomelessness.jpg';
 import ImgAdultHomelessnessPartner from './assets/logos/JDC.png';
 
-const GuideCard = props => (
-  <a className={styles.cardLink} href={props.link}>
-    <div className={styles.card}>
-      <img
-        className={styles.cardImage}
-        src={props.img}
-        alt={props.name}
-      />
-      <div className={styles.cardTextWrapper}>
-        <div className={styles.cardText}>
-          {props.name}
-          <a className={styles.cardLinkText} href={props.link}>
-          Explore Guide →
-          </a>
-        </div>
-        <div className={styles.cardPartner}>
-          <small>In partnership with</small>
-          <img
-            className={styles.cardPartnerImage}
-            src={props.partnerImg}
-            alt={props.partner}
-          />
+function typeform(event, link) {
+  const typeformReference = typeformEmbed.makePopup(
+    link,
+    {
+      mode: 'popup',
+      hideFooters: true,
+    },
+  );
+  typeformReference.open();
+}
+
+const GuideCard = props => {
+  const { name } = props;
+  const { img } = props;
+  const { link } = props;
+  const { partner } = props;
+  const { partnerImg } = props;
```

Comment:
Nit: My preference is to do all this in a single destructuring assignment in the function argument definition itself. Something like this: https://github.com/ShelterTechSF/askdarcel-web/blob/099d26fdf4f2395de8cd1c2aeddca25a57ba07d9/app/components/ui/HamburgerMenu.jsx#L37-L44

Reply:
I'm going to go ahead and make this change right now, since it seems like we're interested in getting the landing page deployed to staging soon.